### PR TITLE
add a new blog

### PR DESCRIPTION
**Pull request by langyuemeng**
@mahjonp @YiniXu9506 PTAL

### CODE REVIEW

File: use-horoscope-to-test-tidb-optimizer.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+2. 另一类则是在统计信息过期的情况下给错了执行计划。
+
+选错索引是其中比较常见的一种情况，用户希望添加索引来加速查询速度，某些情况下，优化器可能会走到全表扫的物理执行计划或者选错索引使得实际执行效果退化成全表扫的情况。
+
+针对上述情况，我们需要从更微观的层面来度量优化器的执行计划和索引选择的性能，评估在优化器上做的改进工作能否切实起到期望的效果。
+
+## 为什么我们要开发 Horoscope?
+
+为了测量优化器和执行器，从去年开始我们构建了daily benchmark 平台 perf.pingcap.com，覆盖常见的几种复杂查询的测试场景，包含 TPC-H、TPC-DS
```
Comment (mahjonp) [inline use-horoscope-to-test-tidb-optimizer.md:23]:
```suggestion
为了测量优化器和执行器，从去年开始我们构建了daily benchmark 平台 perf.pingcap.com，覆盖常见的几种复杂查询的测试场景，包含 TPC-H、TPC-DS、
```

**Review by mahjonp — COMMENTED** [review]
Rest LGTM

**Review by mahjonp — APPROVED** [review]
lgtm

**Review by YiniXu9506 — APPROVED** [review]
LGTM

**Pull request merged**